client/src/components/pages/not-found/not-found.css

```
@@ -1,3 +1,12 @@
+/* Layout */
 .not-found {
+width: 1440px;
+height: 223px;
+flex-shrink: 0;
+}
 
+/* Style */
+.not-found {
+    background: #2e2e2e;
+    box-shadow: 0px 25px 20px 0px rgba(0, 0, 0, 0.45);
 }
```

You can place all styles in the same declaration block.
ALSO note that I have suggested 100% width instead of 1440px. This will fix the rendering issues on different screen sizes.

```suggestion
/* Layout */
.not-found {
  width: 100%;
  height: 223px;
  flex-shrink: 0;
  background: #2e2e2e;
  box-shadow: 0px 25px 20px 0px rgba(0, 0, 0, 0.45);
}
```